src/main/java/com/deavensoft/paketomat/center/CenterServiceImpl.java

```diff
@@ -1,46 +1,54 @@
 package com.deavensoft.paketomat.center;
 
 import com.deavensoft.paketomat.center.model.Package;
+import com.deavensoft.paketomat.center.model.Status;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 @Service
-public class CenterServiceImpl implements CenterService{
+public class CenterServiceImpl implements CenterService {
     private CenterRepository centerrepository;
```

Comment:
Java follows the camel-case syntax for naming the classes, interfaces, methods, and variables.